test/jdk/java/nio/channels/DatagramChannel/Connect.java

```diff
@@ -114,9 +118,14 @@ public void run() {
                 ByteBuffer bb = ByteBuffer.allocateDirect(MAX);
                 bb.put(bytes);
                 bb.flip();
+                if (Platform.isOSX()) {
```

Comment:
If we look at this test next year then we will wonder why the bind is macOS specific, I think this needs a comment.